terraform/layers/python3.10/geopandas_no_numpy_no_pandas/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM --platform=linux/amd64 public.ecr.aws/lambda/python:3.10
+
+ENV NUMPY_VERSION="1.26.4"
+ENV PANDAS_VERSION="1.5.3"
+
+WORKDIR /opt
+RUN mkdir -p python
+
+RUN pip install "numpy==${NUMPY_VERSION}" "pandas==${PANDAS_VERSION}" -t python
```

Comment:
You're not installing geopandas here, right?  This seems wrong, maybe just a copy of the pandas_no_numpy Dockerfile.